usermanagerlib/src/test/java/org/pesmypetcare/usermanagerlib/clients/PetManagerClientTest.java

```diff
@@ -39,10 +40,19 @@
     private static final String BIRTH_FIELD = "birth";
     private static final String CODE_OK = "Should return response code 200";
     private static final StringBuilder STATUS_OK = new StringBuilder("200");
-    private final double recommendedKcal = 2.5;
+    private static final double RECOMMENDED_KCAL_EXAMPLE = 2.5;
+    private static final String date1 = "1990-01-08T15:20:30";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "1990-01-08T15:20:30" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5479466)